features/cli/pod.feature

```diff
@@ -243,7 +243,7 @@ Feature: pods related scenarios
   @critical
   Scenario: OCP-22283:Authentication 4.0 Oauth provider info should be consumed in a pod
     Given I have a project
-    When I create a new application with:
+    When I create a new application with importmode if supported with:
```

Comment:
@varad-ahirwadkar where is this method defined?

Reply:
Sorry, I did not see it. It is not available under this repository.

Reply:
I see another PR for the same test case https://github.com/openshift/verification-tests/pull/3462.

Reply:
@varad-ahirwadkar, currently, there's no such step_defintion, so you new addition will definitely fail.  Have you tested it with an actual run?  BTW, that PR that you reference is calling a different step than what you are calling.
```
When I create a new application with:
```

Reply:
Yes it fails. Sorry for inconvenience. By mistake I used this method which is not part of this repository.

Reply:
Given PR https://github.com/openshift/verification-tests/pull/3468 is merged, I think the change is no longer needed.

Reply:
Thanks. I am closing this PR.